### ISSUE TITLE
Changing icon colors

### ISSUE THREAD

**Issue by Rubonnek** [hbstack/socials#29]
There doesn't seem to be a way to change or disable the icon colors. Is that right?

**Comment by razonyang** [comment]
yep, currently, the related modules (header/modules/socials and footer/modules/socials) do not support disable or change icon's colors.

but I could add some parameters to disable _all_ social icons colors, not sure if it fit your need.but I could add some parameters to disable _all_ social icons colors, not sure if it fit your need.

PR are welcome if you think some icon color are incorrect.

**Comment by Rubonnek** [comment]
> But I could add some parameters to disable _all_ social icons colors, not sure if it fit your need.

This is actually what I want to do. I'd like to disable the color on all of the icons. I personally find the colored icons add visual noise that I'd like to avoid in my use case.

**Comment by razonyang** [comment]
Available since `github.com/hbstack/footer/modules/socials` `v0.2.0`, in short:

```
params:
  hb:
    footer:
      socials:
        _color: false
        # ...
```

See the [footer socials module](https://hbstack.dev/en/modules/footer/socials/) for details.

The header socials module is non-colorful, no need to change.

**Comment by Rubonnek** [comment]
Thanks! That works great!

I think the author page social icons is the only one missing this feature now. I'm not sure which HB module that one is.

**Comment by razonyang** [comment]
> I think the author page social icons is the only one missing this feature now

Ah, I forgot that, will create a PR later.

**Comment by razonyang** [comment]
Available since `github.com/hbstack/blog` `v0.13.0`, same format as above (author page's front matter).

**Comment by Rubonnek** [comment]
Thank you!
